api_tests/registrations/views/test_registration_detail.py

```diff
@@ -382,7 +382,8 @@ def test_registration_fields_are_read_only(self):
             'registration_choice',
             'lift_embargo',
             'tags',
-            'custom_citation']
+            'custom_citation',
+            'child_of']
```

Comment:
Why is `child_of` being added to a list of "writeable_fields"?

Reply:
After... disinheriting(?) the SerializerMethod class from the FilterOnlyField these changes were no longer necessary to pass this test so I removed them.